### DISCUSSION TITLE
How to save setting layout vscode and use it for another project

### DISCUSSION THREAD

**Discussion by lazarus2019** [microsoft/vscode-discussions#564] [Extension Development QnA] [answered]
This is my expect template layout and I want use this template for every project I open

Can you suggest any extension, trick or settings to make it.

![image](https://user-images.githubusercontent.com/62226062/231929445-2c9bc3d1-a92c-4dbb-baa4-5f3e020e3f6a.png)

**Comment by isidorn** [comment, marked as answer]
Currently not possible. There is probalby already a feature request in VS Code for this. So I am closing this one as a dup. And it is not extension author related.
